tests/test_pixel.py

```diff
@@ -1790,6 +1790,50 @@ def test_convert_func():
         print('fitted params = ',star1.fit.params)
         np.testing.assert_allclose(star1.fit.params, model_im.array.ravel(), rtol=2.e-3)
 
+@timer
+def test_convergence_centering_failed():
+
+    # Before https://rubinobs.atlassian.net/browse/DM-49152, it was
+    # running an infinite number of time. Reason is because fitting center
+    # looks to fail in some cases. So now it should converge if center fail.
+    # It should converged now and exclude properly stars that were not
+    # able to be get a good center fit and then being reflux. If centering,
+    # is fixed, this test might need to be removed.
+
+    piffConfig = {
+        'type': 'Simple',
+        'model': {
+            'type': 'PixelGrid',
+            'scale': 0.2,
+            'size': 25,
+            'interp': 'Lanczos(11)'
+        },
+        'interp': {
+            'type': 'BasisPolynomial',
+            'order': 1,
+            'solver': 'scipy'
+        },
+        'outliers': {
+            'type': 'Chisq',
+            'nsigma': 4.0,
+            'max_remove': 0.05
+        },
+        'max_iter': 30
+    }
+
+    dic = pickle.load(open('input/testStars.pkl', 'rb'))
```

Comment:
Probably best to not use pickle.  This can break randomly.  Use the piff.write functionality to write this to a fits file or something.  That should be more stable.  (And not require numpy version to match -- I'm guessing this is why you thought you needed numpy 2.x.)